dist/api_data.py

```diff
@@ -1295,6 +1295,12 @@ def __cmp__(self, other):
     Config('leak_memory', 'false', r'''
         don't free memory during close''',
         type='boolean'),
+    Config('use_timestamp', 'true', r'''
+        by default, create the close checkpoint as of the last stable timestamp
+        if timestamps are in use, or all current updates if there is no
+        stable timestamp set.  If false, this option generates a checkpoint
+        with all updates including those later than the timestamp''',
```

Comment:
In the sentence, "If false", may be better, not to reference timestamp, like "If false, this option generates a checkpoint with all updates"

Reply:
Fixed. Thanks.